web/frontend/src/app/component/general-search/general-search.component.ts

```diff
@@ -403,6 +404,7 @@ export class GeneralSearchComponent implements OnInit,
       // Remove tabs and quotes from search term
       this.searchCriteria.term = String(this.searchCriteria.term).replace('\t', '');
       this.searchCriteria.term = String(this.searchCriteria.term).replace(/\"/g, '');
+      console.log(this.searchCriteria);
```

Comment:
Peter, do me a favor and remove this logging entry just to keep things clean.

Reply:
done in new commit